server/lib/api/transaction.js

```diff
@@ -132,7 +133,54 @@ module.exports = function transactionAPI(router) {
           }
         });
     } else if (req.query.address) {
+      getBlock(
+        { $or:
+          [
+            { 'txs.outputs.address':     req.query.address },
+            { 'txs.inputs.prevout.hash': req.query.address },
+          ],
+        },
+        { rawBlock: 0 },
+        MAX_BLOCKS,
+        (err, block) => {
+          if (err) {
+            res.status(501).send();
+            logger.log('err', err);
+          }
 
+          if (block[0]) {
+            const b = block[0];
+            res.json({
+              pagesTotal: 1,
+              txs: b.txs.map(tx => ({
+                txid: tx.hash,
```

Comment:
would be good to have a standard transform that turns the persisted (bcoin) representation to the client side consumable form.